src/main/cliapp/src/service/DiseaseAnnotationService.js

```diff
@@ -1,6 +1,10 @@
 import axios from 'axios';
 
 export class DiseaseAnnotationService {
+
+    saveDiseaseAnnotation(newAnnotation){
+        return axios.put(`api/disease-annotation`, newAnnotation)
+    }
```

Comment:
Sorry to be picky... but wouldn't name this variable "newAnnotation" as its not "new" vs a post which would be "new"